lib/agent/getKnowledgeBaseContext.ts

```diff
@@ -0,0 +1,44 @@
+import supabase from "@/lib/supabase/serverClient";
+
+interface KnowledgeBaseEntry {
+  url: string;
+  name: string;
+  type: string;
+  content?: string;
+}
+
+/**
+ */
+export async function getKnowledgeBaseContext(roomId: string): Promise<string> {
+  try {
+    const { data: room } = await supabase
+      .from("rooms")
+      .select("artist_id")
+      .eq("id", roomId)
+      .single();
+    
+    if (!room?.artist_id) return "";
+
+    const { data: accountInfo } = await supabase
+      .from("account_info")
+      .select("knowledges")
+      .eq("account_id", room.artist_id)
+      .single();
```

Comment:
please define all supabase queries as standalone libs in `/lib/supabase` so they can be reused in the future.